Sample Applications/Win11ThemeGallery/MainWindow.xaml.cs

```diff
@@ -27,6 +27,29 @@ public MainWindow(MainWindowViewModel viewModel, IServiceProvider serviceProvide
         DataContext = this;
         InitializeComponent();
 
+        var appContextBackdropData = AppContext.GetData("Switch.System.Windows.Appearance.DisableFluentThemeWindowBackdrop");
+        bool disableFluentThemeWindowBackdrop = false;
```

Comment:
Move this code in a seperate method, which determines the visibility of buttons and set he visibility here

Reply:
Addressed